geopy/geocoders/geonames.py

```diff
@@ -87,6 +95,12 @@ def __init__(
         self.api_reverse = (
             "%s://%s%s" % (self.scheme, domain, self.reverse_path)
         )
+        self.tz_api = (
+            "%s://%s%s" % (self.scheme, domain, self.timezone_path)
+        )
+        self.nearby_api = (
```

Comment:
The naming style of these attributes should match the already existing one: `api_reverse` has `api` part in the beginning of the name instead of the end.